### PR TITLE
Added labels to output

### PR DESCRIPTION
**Pull request by DenverCoder1**
I added a list of labels in the output.

When deciding whether to look at an issue, the labels may be helpful.

image

### All Submissions:

* [x] Does your PR have the correct target branch?
* [x] Have you followed the guidelines for [contributing](../#how-to-contribute)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/<User>/<Repository>/pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by icanhazstring — APPROVED** [review]
Nice addition thank you 🙏

**Comment by icanhazstring** [comment]
There are some checks failing. If you resolved them we can merge 👍

File: src/Model/IssueModel.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ class IssueModel
     private $state;
 
     /**
-     * @var array<int,string>
+     * @var array<int,array<string>>
```
Comment (icanhazstring) [inline src/Model/IssueModel.php]: I think `array<int, array<string, mixed>` would be more correct here

Comment (icanhazstring) [reply src/Model/IssueModel.php]: And just add `array<string>` as return annotation to the `getLabels` method. Should work then :)

Comment (DenverCoder1) [reply src/Model/IssueModel.php]: @icanhazstring I'm getting a syntax error when I try specifying the type for getLabels().
image

Comment (icanhazstring) [reply src/Model/IssueModel.php]: Php itself only allows return type to be of `array`. You need to add an annotation to that.

```
/**
 * @return array<string>
 */
```

Comment (DenverCoder1) [reply src/Model/IssueModel.php]: Thanks! Should be good now.

**Review by icanhazstring — APPROVED** [review]

**Comment by icanhazstring** [comment]
Thanks. Merged 👍

**Pull request merged**